src/main/java/controllers/SwingBoard.java

```diff
@@ -0,0 +1,234 @@
+package controllers;
+
+import entities.ComputerShip;
+import service.GameState;
+import entities.Guess;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.BorderLayout;
+import java.awt.Color;
+import java.awt.event.*;
+import java.util.ArrayList;
+import java.util.logging.Logger;
+import javax.swing.JFrame;
+import javax.swing.JPanel;
+
+public class SwingBoard extends JPanel implements MouseListener {//inheriting JFrame
+    PlayerSelection playerSelection;
+    GameState state;
+    static Logger log = Logger.getLogger("Panel Location");
+
+
+    private ArrayList<Point> computerPositions = new ArrayList<Point>();
+    private JPanel mainPanel, footerPanel;
+    private JFrame frame;
+    private final int LENGTH = 10;
```

Comment:
Consider making these `private static final`, or change the variable name casing to fit java semantics